Source/projections/Store/ProjectionOf.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import { Constructor } from '@dolittle/types';
+import { Cancellation } from '@dolittle/sdk.resilience';
+import { IProjectionStore } from './IProjectionStore';
```

Comment:
```suggestion
import { Cancellation } from '@dolittle/sdk.resilience';

import { IProjectionStore } from './IProjectionStore';
```
My preferred pattern again...